setup.py

```diff
@@ -12,7 +12,7 @@
 
 setup(
     name='drf-extra-fields',
-    version='3.0.4',
+    version='3.0.5',
```

Comment:
It seems like changing `ContentFile` to `SimpleUploadedFile` is backward incompatible. `ContentFile` overrides following methods: `__init__`, `__str__`, `__bool__`, `open`, `close` and `write`. These methods also exist in `SimpleUploadedFile` but they are slightly different in two classes, so it may cause bugs.

Considering this, should we update the major version, or changing minor version is enough? I think changing patch version (like in this commit) is erroneous. What do you think? @alicertel

Reply:
Let's go with 3.1.0 and document this breaking change in the beginning of the readme. What do you think?

Reply:
I totally agree. I have made the required changes:

a1cb26a
ce1a970

Reply:
Thank so much. I will take a new release soon.